app/views/definition.scala.txt

```diff
@@ -0,0 +1,20 @@
+@import play.api.libs.json.Json
+@(apiContext: String)
+{
+  "scopes":[],
+  "api": {
+    "name": "Check a UK VAT number",
+    "description": "Allows for retrieval VAT registered companies",
```

Comment:
if you've got a user on the browser you'll eventually want to get this description checked and written by your content designer - it'll appear on the screen in front of the user in the grant permission page

Reply:
Yes - I'll raise a ticket.